Events/Ch16.event

```diff
@@ -0,0 +1,180 @@
+UNIT Knoll Druid 0x00 Level(6, Ally, 1) [17,19] 0x00 0x00 0x0 0x00 [Flux,Luna,Vulnerary,Vulnerary] NoAI 
+UNIT*/
+
+MariyaGroup:
+UNIT 0x15 WyvernKnight_F 0x00 Level(1, Ally, 0) [3,19] 0x00 0x00 0x1 MariyaREDA [ShortSpear,IronLance,PureWater] NoAI
+UNIT
+MariyaREDA:
+REDA [3,19] 0x0 0x0 0x0 0x0
+
+TurnBasedEvents:
+END_MAIN
+
+CharacterBasedEvents:
+CHAR 0xC MariyaTalk [0x15,0x1] 0x0
+CHAR 0xC MariyaTalk [0x1,0x15] 0x0
+END_MAIN
+
+LocationBasedEvents:
+Chest(MasterSeal,13,9)
+Chest(DelphiShield,5,2)
+Door(6,9)
+Door(7,9)
+Door(4,15)
+Door(5,15)
+END_MAIN
+
+MiscBasedEvents:
+CauseGameOverIfLordDies
+DefeatBoss(EndingScene)
+AREA 0xB MariyaLoad [0,0] [8,19]
+END_MAIN
+
+Dunno:
+//DO NOT TOUCH
+WORD $00
+
+Tutorial:
+//DO NOT TOUCH
+WORD $00
+
+TrapData:
+END_MAIN
+
+ALIGN 4
+
+BeginningScene:
+MUSS 0x4D
+//LOAD2 0x0 Units
+LOAD1 0x0 Units
+ENUN
+LOAD1 0x1 Enemies
+ENUN
+SVAL 0x2 0x34
+SVAL 0x3 BeginningTextCh16
+CALL 0x9EE310
+REMA
+GotoPrepScreen
+ENDA
+
+EndingScene:
+MUSS 0x32
+CAM1 0x1
+CUMO 0x1
+STAL 60
+CURE
+Text(VedgeWinsCh16)
+FADI 4
+SVAL 0x2 0x2C
+CHECK_ALIVE 0x15 //checks if mariya is alive
+BEQ 0x1400 0xC 0x0 //if not alive, send to label 1400
+SVAL 0x3 MariyaAliveCh16
+GOTO 0x3531
+LABEL 0x1400
+SVAL 0x3 MariyaDEADCh16
+LABEL 0x3531
+CALL 0x9EE310
+REMA
+MNC2 0x11
+EVBIT_T 0x7
+ENDA
+
+MariyaLoad:
+SVAL 0x2 0x0
+CALL 0x9EE1B8
+LOAD1 0x1 MariyaGroup
+_MUSICSOMETHING 32767
+MUSS 0x30
+CAM1 0x15
+CUMO 0x15
+STAL 60
+CURE
+Text(MariyaAppearsCh16)
+REMA
+EVBIT_T 0x7
+ENDA
+
+MariyaTalk:
+MUSS 0x30
+Text(MariyaVedgeTalkCh16)
+REMA
+EVBIT_T 0x7
+ENDA
+
+// Events
+
+// Manual Movement
+
+// Scripted Fights
+
+// Units
+Enemies:
+UNIT 0x60 Ranger 0x00 Level(10, Enemy, 1) [16,1] 0x00 0x00 0x0 0x00 [Lancereaver,Runesword,RunicBow] [0x3,0x3,0x4,0x20]
+UNIT 0xa0 Fighter 0x00 Level(17, Enemy, 1) [10,19] 0x0 0x0 0x0 0x0 [SteelAxe] [0x0,0x0,0x0,0x0]
+UNIT 0xa0 Mage 0x00 Level(18,Enemy,1) [9,17] 0x0 0x0 0x0 0x0 [Thunder] [0x0,0x0,0x0,0x0]
+UNIT 0xa0 Sniper 0x0 Level(1,Enemy,1) [6,18] 0x0 0x0 0x0 0x0 [IronBow,Longbow] [0x0,0x0,0x0,0x0]
+UNIT 0xa0 General 0x0 Level(2,Enemy,1) [5,16] DropItem 0x0 0x0 0x0 [HandAxe,SteelLance,DoorKey] [0x3,0x3,0x4,0x20]
+UNIT 0xa0 Myrmidon 0x0 Level(16,Enemy,1) [1,18] 0x0 0x0 0x0 0x0 [SteelSword] [0x0,0x3,0x4,0x0]
+UNIT 0xa0 Bishop 0x0 Level(1,Enemy,1) [2,17] 0x0 0x0 0x0 0x0 [Lightning,Mend] [0xF,0x3,0x4,0x0]
+UNIT 0xa0 Cavalier 0x0 Level(18,Enemy,1) [6,10] 0x0 0x0 0x0 0x0 [Lancereaver,SteelLance] [0x0,0x6,0x0,0x0]
+UNIT 0xa0 Cavalier_F 0x0 Level(17,Enemy,1) [7,10] 0x0 0x0 0x0 0x0 [SteelSword,Javelin] [0x0,0x6,0x0,0x0]
+UNIT 0xa0 General 0x0 Level(2,Enemy,1) [0,12] DropItem 0x0 0x0 0x0 [Tomahawk,DoorKey] [0x3,0x3,0x4,0x20]
+UNIT 0xa0 Shaman 0x0 Level(18,Enemy,1) [1,10] 0x0 0x0 0x0 0x0 [Flux] [0x0,0x3,0x4,0x0]
+UNIT 0xa0 Pirate 0x0 Level(19,Enemy,1) [4,11] DropItem 0x0 0x0 0x0 [SteelAxe] [0x0,0x3,0x4,0x0]
+UNIT 0xa0 Soldier 0x0 Level(20,Enemy,1) [6,12] 0x0 0x0 0x0 0x0 [SteelLance] [0x0,0x3,0x4,0x0]
```

Comment:
Since Debuff data gets overwritten if large unit locks are loaded, please separate this block into to smaller blocks and just load them right after another.